site/content/sponsors/informedk12.md

```diff
@@ -2,5 +2,5 @@
 title: InformedK12
 description: Easy-to-use workflow automation solution that helps school district administrators digitize forms, automate processes and track approvals.
 link: https://www.informedk12.com/careers
-image: https://www.informedk12.com/hubfs/Logo%20%5BJune-17%5D/mark-rgb-isolated.png?t=1536235567849
+image: "/assets/img/sponsor/informedk12.png"
```

Comment:
I see this image added in! 👍

Reply:
Hmm interesting, this image is located in the `dist/assets/img` folder where the remaining images are currently hosted. I'll go ahead and double check the images are being referenced currently.